app/grandchallenge/core/signals.py

```diff
@@ -93,6 +100,8 @@ def update_editor_follows(  # noqa: C901
             follow_objects.append(group.editors_of_archive)
         elif hasattr(group, "editors_of_readerstudy"):
             follow_objects.append(group.editors_of_readerstudy)
+        elif hasattr(group, "admins_of_challenge"):
+            follow_objects.append(group.admins_of_challenge)
```

Comment:
Just a note: we have to be really careful now with what objects the user follows as the challenge now has things that only the admin can see. Could you add a comment about this?

Reply:
So you mean a comment along the lines of "only admins of challenges should follow a challenge" to make sure we don't accidentally create follows for participants at some later point in time?